test/cql-pytest/test_tablets.py

```diff
@@ -127,3 +127,50 @@ def verify_tablets_presence(expected:bool=True):
 
     cql.execute(f"DROP TABLE {test_keyspace}.{table_name}")
     verify_tablets_presence(expected=False)
+
+
+def _test_tablets_are_dropped_when_dropping_table_with_view(cql, keyspace_name, attempt_drop_table:bool=False):
```

Comment:
Please consider using pytest's "paramterize" instead of this ugly underscore idiom from dtest :-(

Reply:
I thought parametrize is a dtest idiom not a generic one

Reply:
It's the opposite - dtest always used that underscore thing, and parametrize was only recently adopted after dtest moved to pytest. You can see in test/cql-pytest quite a few examples of parametrize. It's useful for exactly this case - when you want to run the same test once with true and once with false, or some other case of several runs. pytest also makes it easy to run just one of the parameterizations (e.g., `test/cql-pytest/run somefile.py::sometest[dog]` runs the "dog" case) and they are nicely reported in the output and counted as separate tests.

Reply:
used in v3